amethyst_rendy/src/camera.rs

```diff
@@ -435,12 +516,8 @@ impl Projection {
         screen_diagonal: Vector2<f32>,
         camera_transform: &Transform,
     ) -> Point2<f32> {
-        let render_matrix: Matrix4<f32> =
-            amethyst_core::math::convert(*camera_transform.global_matrix());
-
-        let f = render_matrix * self.as_matrix();
-
-        let screen_pos = f.transform_point(&world_position);
+        let screen_pos =
+            (camera_transform.global_matrix() * self.as_matrix()).transform_point(&world_position);
```

Comment:
You need to bump the nalgebra requirement to `0.18.1` to ensure that `transform_point` returns correct value for everyone.